scripts/deployUnmockedColonyVersions.js

```diff
@@ -0,0 +1,72 @@
+// Input:
+/* globals artifacts */
+
+// Check out v3
+// Deploy v3 resolver
+// Point v3 on network to that resolver
+
+// Checkout out v4
+// Deploy v4 resolver
+// Point v4 on network to that resolver
+const util = require("util");
+const exec = util.promisify(require("child_process").exec);
+const cnAddress = require("../etherrouter-address.json").etherRouterAddress; // eslint-disable-line import/no-unresolved
+
+const IColonyNetwork = artifacts.require("./IColonyNetwork");
+const IMetaColony = artifacts.require("./IMetaColony");
+
+module.exports = async () => {
+  // While debugging, always checkout currenthash
+  // await exec("git checkout 2690fba3d3002fd72912cd74f1fbf4932c734e94 && git submodule update");
+  const currentHash = await exec("git log -1 --format='%H'");
+  // const currentHash = res.stdout.trim();
+  let v3ResolverAddress;
+  let v4ResolverAddress;
+  const colonyNetwork = await IColonyNetwork.at(cnAddress);
+  const metaColonyAddress = await colonyNetwork.getMetaColony();
+  const metaColony = await IMetaColony.at(metaColonyAddress);
+  try {
+    await exec("mv ./build ./buildBackup");
+    await exec("git checkout ad5569de24567517aa12624e29600c9136fb594d && git submodule update");
+
+    // Comment out uneeded parts of file
+    await exec("sed -i'' -e '26,27 s|^|//|' ./migrations/4_setup_colony_version_resolver.js");
+    await exec("sed -i'' -e '31 s|^|//|' ./migrations/4_setup_colony_version_resolver.js");
+    await exec("sed -i'' -e 's|await ContractRecovery.deployed()|await ContractRecovery.new()|' ./migrations/4_setup_colony_version_resolver.js");
```

Comment:
I don't love how brittle this is...

Here's a suggestion: put this logic inside a conditional, and simply `sed` a constant, something along the lines of `const skipForBackVersion = true;` which would then bypass these lines.

Reply:
If I understand your suggestion correctly, you're talking about adding that to the migration file? That's not really possible - the file exists in the git history, I can't really go back and add to it without rebasing everything that's been on `master` since then. I would argue it is not brittle for that reason though - we are checking out a specific commit, so this will never change.

Reply:
Ah yes fair, this isn't the `develop` migrations. Carry on 🚀